src/main.cpp

```diff
@@ -632,7 +632,7 @@ int main ( int argc, char** argv )
     QCoreApplication* pApp = new QCoreApplication ( argc, argv );
 #else
 # if defined ( Q_OS_IOS )
-    bIsClient = false;
+    bIsClient = true;
```

Comment:
Probably we'd have to think how to change this during runtime or if we want to be able to change it. But that's not part of this issue.

Reply:
for now, I cannot image why on iOS you would want to run in server mode. Plus - how to use the command line switches?

Reply:
Not sure. I think you can even remove it since it's true by default

Reply:
If I may: when the app works on iOS, I'd love to be able to start a server on my iPad, so I can be more mobile. But that'll require some more work to separate server and client modes.

Reply:
See line 60: I think we can remove this one.